homeassistant/components/light/hue.py

```diff
@@ -310,8 +327,11 @@ def device_info(self):
             # productname added in Hue Bridge API 1.24
             # (published 03/05/2018)
             'model': self.light.productname or self.light.modelid,
+            'model_id': self.light.modelid,
             # Not yet exposed as properties in aiohue
             'sw_version': self.light.raw['swversion'],
+            'color_gamut_type': light_spec['colorgamuttype'],
+            'color_gamut': light_spec['colorgamut'],
```

Comment:
The device registry has specific attributes, you should not just add your own stuff. https://developers.home-assistant.io/docs/en/device_registry_index.html